### DISCUSSION TITLE
Crane + Fenix + WASM

### DISCUSSION THREAD

**Discussion by caniko** [nix-community/fenix#190] [Q&A]
Hello, could one if you lovely people share an example with Crane + Fenix + WASM?

I have the crane WASM example already, but there is no Fenix integration.

**Comment by eureka-cpu** [comment]
Hi @caniko have a look at my draft PR: #239

**Reply by eureka-cpu** [reply]
There will eventually be some template examples. This one isn't using flakes but I think you'll still get what you need from it.
